### DISCUSSION TITLE
The screen in blank

### DISCUSSION THREAD

**Discussion by jesseflint** [mozilla/hubs#3051] [General Questions & Discussion] [answered]
Enter a room.
Move around the room
See error, the screen goes completely blank but I can still hear audio
Close tab, close chrome, reopen Chrome, start new tab, enter room again
Still see nothing on the screen

![Issue](https://user-images.githubusercontent.com/54375935/93641966-433d7580-f9cb-11ea-92b1-c6d8f365ef8b.png)

**Hardware** 
 - Device: Desktop
 - OS: Windows 10
 - Browser: hrome

**Comment by johnshaughnessy** [comment]
Can you include a log from the Chrome web console? It may contain a relevant error message. (Instructions for opening the console: https://developers.google.com/web/tools/chrome-devtools/open#console )

Does this error occur in the default scene that comes with a new room created on https://hubs.mozilla.com/ ? If not, it might be an issue with the scene file. In that case it might help to link the scene for further debugging.

Does this same error occur in a private browser window with all extensions disabled? If not, there might be some interference with other extensions or something in local storage.

**Comment by misslivirose** [comment, marked as answer]
I am able to enter the room that is shown in this screenshot - scene also loads fine. I'm going to convert this to a discussion and when we have additional information, we can file a new issue.

**Reply by robinkwilson** [reply]
Please mark as unanswered if you post more information on this. Marking this as answer to get this out of our pipeline.
